ltp-production.yaml

```diff
@@ -1195,13 +1195,10 @@ projects:
     url: https://bugs.linaro.org/show_bug.cgi?id=5640
     active: true
     intermittent: true
-  - environments:
-    - juno-r2
-    - juno-r2-kasan
-    - juno-r2-compat
-    - juno-64k_page_size
+  - environments: *environments_all
```

Comment:
when did these start to fail on all devices?
And no one have replied back to your reports upstream?

Reply:
Test case is failing on one of those environments is expected to fail on future devices / environments. It is not easy to maintain the list of environments update manually.

Reports are getting noise due to this test case added to limited number of environments as a known issue.